corehq/apps/geospatial/static/geospatial/js/models.js

```diff
@@ -692,7 +695,7 @@ hqDefine('geospatial/js/models', [
                     }, 2000);
                 },
                 error: function () {
-                    alertUser.alert_user(gettext(unexpectedErrorMessage), 'danger');
```

Comment:
Hey @mkangia ,
I was wrong about this when I mentioned this `gettext(unexpectedErrorMessage)` works.  The alert message is shown correctly however the text is not picked up for translation. 
It is now fixed in this commit.